packages/ia-components/index.less

```diff
@@ -3,7 +3,7 @@
 @import './live/example-react-component/styles.less';
 
 @import './sandbox/theatres/with-youtube-spotify/audio-with-youtube-spotify.less';
-@import './sandbox/theatres/components/track-list/track-list.less';
-@import './sandbox/theatres/components/audio-player/audio-player.less';
+@import './sandbox/theatres/track-list/track-list.less';
+@import './sandbox/theatres/audio-player/audio-player.less';
```

Comment:
same for these:
`/sandbox/track-lists/track-list.less`
`/sandbox/audio-players/audio-player.less`